spring/integration/pubsub/pom.xml

```diff
@@ -0,0 +1,130 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why XML configs should be with the License as well?
Is that your policy?
We in Spring only care about classes (Java/Groovy/Kothlin etc.)